redux/apis/api-types.ts

```diff
@@ -42,7 +45,7 @@ export interface ResendEmailVerificationLinkResponse {
   };
 }
 
-export interface VerifyEmailResponse {
+export interface AuthenticationResponse {
```

Comment:
keep the interface name similar to the api created in auth-api.ts. Either change the name of api in that file or change it here

Reply:
@imrankhan-stixor , We don't need to create redundant types, we can use generic name for types and use it for multiple api's if the response is same.

Reply:
okay I didn't know you're reusing it so good scene